internal/services/recoveryservices/recovery_services_vault_resource.go

```diff
@@ -286,72 +296,13 @@ func resourceRecoveryServicesVaultCreate(d *pluginsdk.ResourceData, meta interfa
 		return fmt.Errorf("creating %s: %+v", id.String(), err)
 	}
 
-	// `encryption` needs to be set before `cross_region_restore_enabled` is set. Or the service will return an error. "If CRR is enabled for the Vault, the storage state will be locked and it will interfere with further operations"
-	// recovery vault's encryption config cannot be set while creation, so a standalone update is required.
```

Comment:
this seems like a pretty important behavioural change - has this codepath been tested?

Reply:
In the past version, these settings should be set by a different API, it can not be set by the "Vaults" API itself. So it requires an additional update for it. But for this new API version it could be done by 1 API. And it has been tested with `TestAccRecoveryServicesVault_CrossRegionRestoreWithEncryption` test case